index.bs

```diff
@@ -859,33 +858,36 @@ partial interface HTMLIFrameElement {
     </div>
   </section>
   <section>
-    ## <dfn export abstract-op id="create-for-browsingcontext">Create a Permissions Policy for a browsing context</dfn> ## {#algo-create-for-browsingcontext}
+    ## <dfn export abstract-op id="create-for-navigable">Create a Permissions Policy for a navigable</dfn> ## {#algo-create-for-navigable}
 
-    <div class="algorithm" data-algorithm="create-for-browsingcontext">
-    Given a <a>browsing context</a> (|browsingContext|), and an <a>origin</a>
+    <div class="algorithm" data-algorithm="create-for-navigable">
+    Given null or an <a for="/">element</a> (|container|) and an <a>origin</a>
     (|origin|) this algorithm returns a new <a>Permissions Policy</a>.
+    1. Assert: If not null, |container| is a <a>navigable container</a>.
     1. Let |inherited policy| be a new ordered map.
     1. Let |declared policy| be a new ordered map.
     1. For each |feature| supported,
         1. Let |isInherited| be the result of running <a abstract-op>Define an
-          inherited policy for feature in browsing context</a> on |feature|,
-          |origin| and |browsingContext|.
+          inherited policy for feature in container at origin</a> on |feature|,
+          |container| and |origin|.
         1. Set |inherited policy|[|feature|] to |isInherited|.
     1. Let |policy| be a new <a>permissions policy</a>, with inherited policy
       |inherited policy| and declared policy |declared policy|.
     1. Return |policy|.
 
+    Note: HTML should be updated to call this algorithm from "create a new
+    browsing context and document".
```

Comment:
This shouldn't be necessary, hopefully, since we'll just merge the two PRs in close proximity.

Reply:
Removed!